apps/src/gamelab/AnimationPicker/actions.js

```diff
@@ -12,9 +12,11 @@ var AnimationPickerAction = module.exports.AnimationPickerAction = {
  * Show the animation picker
  * @returns {{type: AnimationPickerAction}}
  */
-module.exports.showAnimationPicker = function () {
+module.exports.showAnimationPicker = function (onComplete, onCancel) {
   return {
-    type: AnimationPickerAction.SHOW_ANIMATION_PICKER
+    type: AnimationPickerAction.SHOW_ANIMATION_PICKER,
+    onComplete: onComplete,
+    onCancel: onCancel
```

Comment:
Putting functions inside of our reducer/store has always felt a little wrong to me, but I'm not too sure exactly why. This SO article provides some reasonsing http://stackoverflow.com/questions/35325195/should-i-store-function-references-in-redux-store essentially, because then you can't easily serialize your state. This doesn't necessarily matter for us, since we're not transferring this state between client and server at any point, but it does mean it's at least somewhat harder to look at the store at any particular point in time and know exactly what the state of the world is.

Couldn't we just have an unconnected AnimationPicker, and then a ConnectedAnimationPicker that grabs the actions (without them being in the store) and passes them to AnimationPicker?